pyproject.toml

```diff
@@ -62,10 +62,6 @@ Documentation = "https://ddtrace.readthedocs.io/en/stable/"
 Homepage = "https://github.com/DataDog/dd-trace-py"
 "Source Code" = "https://github.com/DataDog/dd-trace-py/"
 
-[tool.setuptools_scm]
```

Comment:
you can also remove from above under `[build-system]`, and I think it is `setup.py` as well?

Reply:
removing it was breaking things. It looks like either it's used for some thing else, or possibly installing it is also installing other things that we need.

Reply:
🤔